components/GlobalSpinner/GlobalSpinner.tsx

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Gridiron Survivor.
+// Licensed under the MIT License.
+
+import React from 'react';
+import Image from 'next/image';
+
+/**
+ * GlobalSpinner component
+ * @returns {React.JSX.Element} GlobalSpinner component
+ */
+const GlobalSpinner = (): React.JSX.Element => (
+  <div
+    className="global-spinner w-full h-full grid place-content-center"
+    data-testId="global-spinner"
+  >
+    <Image
+      alt="Gridiron Survivor logo fading in and out"
+      className="animate-pulse h-64 w-64"
+      data-testId="global-spinner-image"
+      height={100}
+      src="../../assets/global-spinner.svg"
+      width={100}
```

Comment:
Thought: In my most recent [PR](https://github.com/LetsGetTechnical/gridiron-survivor/pull/454/), I had to modify `src="/assets/global-spinner.svg"` so that the logo would render. I'm not sure if you need to do it on yours

Reply:
@vmaineng this should be fixed in the latest version of this component. Please pull and try again, I don't believe you'll have to change anything.